opentracing-jdbc/opentracing-jdbc/src/main/java/org/zalando/opentracing/jdbc/span/Peer.java

```diff
@@ -0,0 +1,31 @@
+package org.zalando.opentracing.jdbc.span;
+
+import lombok.AllArgsConstructor;
+import lombok.Generated;
+import lombok.Getter;
+import lombok.experimental.Wither;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@AllArgsConstructor(access = PRIVATE)
+@Getter
+final class Peer {
+
+    static final Peer EMPTY = new Peer(null, null, null, null, null);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/tracer/pullRequest?prid=4563925)